audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
> I know having two clones of the unit (and preventing their availability based on factions) is bad but taht's how RA2 circumvented that issue.

Just no, :-1:  to cloning the Engineer (also the Dog). We should do a code wise fix for it.

What do game do if a sound file not found? Crash, not play anything or just ignore it and play another one? Because it looks like game will search for non-existing "ienssed" from current yaml code in this PR.